controllers/clusterurlmonitor/clusterurlmonitor_supplement.go

```diff
@@ -25,6 +25,20 @@ const (
 
 // Takes care that right PrometheusRules for the defined ClusterURLMonitor are in place
 func (s *ClusterUrlMonitorReconciler) EnsurePrometheusRuleExists(clusterUrlMonitor v1alpha1.ClusterUrlMonitor) (utilreconcile.Result, error) {
+	// If .spec.skipPrometheusRule is true, ensure that the PrometheusRule does NOT exist
+	if clusterUrlMonitor.Spec.SkipPrometheusRule {
```

Comment:
Will `SkipPrometheusRule` have a default value? Or is it expected to be set on all clusterUrlMonitors?

If there's not a default value, it might be good to confirm that there's a `SkipPrometheusRule` present in the Spec to avoid a null pointer exception

Reply:
It is! It's defaulted as false https://github.com/openshift/route-monitor-operator/pull/223/files#diff-1f765eef49a09245f809b9f25819cb33df4c1be9421a3f735cdb96d8104efe80R41-R42

Reply:
Sorry, I overlooked the kubebuilder stuff and only looked at the json definition!

/lgtm